README.md

```diff
@@ -1,53 +1,129 @@
+Before trying to run you need to aquire token from [Discord](https://discord.com/developers/applications).
+
+Project supports use of [`Seq`](https://datalust.co/seq) server which can be run locally or remotly for storing logs.
+
+### Docker run <a id="tc13"></a>
+
+Here steps for use of deployable version of DiscordBot. Default Docker Hub image could be found [here](https://hub.docker.com/r/seker212/discord_bot)
+
+#### Enviroment informations <a id="tc131"></a>
+
+Running in docker requires setting important evnviromental variables as well as some volumes for files that should be accessible after shutdown of container.
+
+Variables marked with `Seq server` are referenced to different image for running Seq toghether with DiscordBot
+
+Enviroment variables:
+
+| Name |Value|Required | Purpose  |
+|----------|:-------------:|:----:|----|
+| TOKEN |  string (long chain) | X |  Discord bot token can't be run without it. |
+| SEQ_URL |  string (url) |  | Seq application url, local or remote |
+| ACCEPT_EULA |  Y or N (single char) |  | `Seq server` container acceptance of eula if run with DiscordBot together|
+
+Volumes:
+
+| Name |      Container Path      | Required | Purpose |
+|----------|:-------------:|:----:|-----|
+| Audio |  `/app/audio` | X | All sounds files are stored here |
+| Data |  `/app/data` | | Database file is stored here with all servers configurations |
+| Seq/logs |  `/data` | | `Seq server` container data folder for logs storage if run with DiscordBot together|
+
+#### Using docker cli  <a id="tc132"></a>
+
+Pulling and running container could be done using command below, simply place the TOKEN and name, and set the volume path in PWD.
+
+```bash
+docker run --rm -d -e "TOKEN=<discord app client secret here>" --name <name> -v <PWD>/audio:/app/audio seker212/discord_bot:2.0
+```
+
+#### Using docker compose <a id="tc133"></a>
+
+Use provided [`docker-compose-template.yaml`](docker-compose-template.yaml) and rename it to `docker-compose.yaml` or based on example below and enviroment variables create your own `docker-compose.yaml` file. Replace required values and run by using `docker compose up -d` command for Docker v2 or `docker-compose up -d` for v1.
 
 ```yaml
 
 version: "3.9"
 services:
-  Chi-chan:
+  DiscordBot:
     image: seker212/discord_bot:2.0
     environment:
       # Set this a discord token generated on discord page
       - TOKEN=0
     volumes: 
-      # Where audio files from .sound commands should be placed
+      # Where audio files from sound commands should be placed
       - ./audio:/app/audio
 
 ```
 
-To run it simply execute docker compose command, depending on compose engine installed on your system.
+### Local run <a id="tc12"></a>
 
-```bash
-# V1
-docker-compose up -d
-# V2
-docker compose up -d
-```
+Running locally is not advised as it is recommended to use of Visual Studio and installation of .Net 6. For debbuging it is prefered method.
```

Comment:
What do you mean you "don't advide it, because it's recommended to use VS"? This sentence is tragic.